docs/advanced_examples/ClientServer.ipynb

```diff
@@ -232,10 +232,7 @@
     "fhemodel_client = FHEModelClient(network.client_dir.name, key_dir=network.client_dir.name)\n",
     "\n",
     "# The client first need to create the private and evaluation keys.\n",
-    "fhemodel_client.generate_private_and_evaluation_keys()\n",
-    "\n",
-    "# Get the serialized evaluation keys\n",
-    "serialized_evaluation_keys = fhemodel_client.get_serialized_evaluation_keys()"
+    "serialized_evaluation_keys = fhemodel_client.generate_private_and_evaluation_keys()"
```

Comment:
this line should still use `get_serialized_evaluation_keys`

Reply:
Good catch. updated.